ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -221,9 +221,9 @@ StackView {
         SettingsPage {
             id: ownerTokenPage
 
-            property int chainId
-            property string accountName
-            property string accountAddress
+            property int chainId: SQUtils.ModelUtils.getByKey(root.flatNetworks, "layer", 2).chainId
```

Comment:
Not wrong, but these should imo go to the line 213 above, as an object, like it's done on line 134:

```qml
function retryPrivilegedToken(key, chainId, accountName, accountAddress) {
            var properties = {
                key: key,
                chainId: chainId,
                accountName: accountName,
                accountAddress: accountAddress,
            }

            root.push(ownerTokenEditViewComponent, properties,
                      StackView.Immediate)
        }
```

Reply:
Explains why the "retry" worked while the initial didn't :)

Reply:
Good point, updated.